src/containerapp/azext_containerapp/commands.py

```diff
@@ -0,0 +1,107 @@
+        g.custom_command('create', 'create_managed_environment', supports_no_wait=True, exception_handler=ex_handler_factory())
+        g.custom_command('delete', 'delete_managed_environment', supports_no_wait=True, confirmation=True, exception_handler=ex_handler_factory())
+
+    with self.command_group('containerapp env dapr-component') as g:
+        g.custom_command('list', 'list_dapr_components')
+        g.custom_show_command('show', 'show_dapr_component')
+        g.custom_command('set', 'create_or_update_dapr_component')
+        g.custom_command('remove', 'remove_dapr_component')
+
+    with self.command_group('containerapp identity') as g:
+        g.custom_command('assign', 'assign_managed_identity', supports_no_wait=True, exception_handler=ex_handler_factory())
+        g.custom_command('remove', 'remove_managed_identity', supports_no_wait=True, exception_handler=ex_handler_factory())
+        g.custom_show_command('show', 'show_managed_identity')
+
+    with self.command_group('containerapp github-action') as g:
+        g.custom_command('add', 'create_or_update_github_action', exception_handler=ex_handler_factory())
+        g.custom_show_command('show', 'show_github_action', exception_handler=ex_handler_factory())
+        g.custom_command('delete', 'delete_github_action', exception_handler=ex_handler_factory())
+
+    with self.command_group('containerapp revision') as g:
+        g.custom_command('activate', 'activate_revision')
+        g.custom_command('deactivate', 'deactivate_revision')
+        g.custom_command('list', 'list_revisions', table_transformer=transform_revision_list_output, exception_handler=ex_handler_factory())
+        g.custom_command('restart', 'restart_revision')
+        g.custom_show_command('show', 'show_revision', table_transformer=transform_revision_output, exception_handler=ex_handler_factory())
+        g.custom_command('copy', 'copy_revision', exception_handler=ex_handler_factory())
+        g.custom_command('set-mode', 'set_revision_mode', exception_handler=ex_handler_factory())
+
+    with self.command_group('containerapp ingress') as g:
+        g.custom_command('enable', 'enable_ingress', exception_handler=ex_handler_factory())
+        g.custom_command('disable', 'disable_ingress', exception_handler=ex_handler_factory())
+        g.custom_show_command('show', 'show_ingress')
+
+    with self.command_group('containerapp ingress traffic') as g:
+        g.custom_command('set', 'set_ingress_traffic', exception_handler=ex_handler_factory())
+        g.custom_show_command('show', 'show_ingress_traffic')
+
+    with self.command_group('containerapp registry') as g:
+        g.custom_command('set', 'set_registry', exception_handler=ex_handler_factory())
+        g.custom_show_command('show', 'show_registry')
+        g.custom_command('list', 'list_registry')
+        g.custom_command('remove', 'remove_registry', exception_handler=ex_handler_factory())
```

Comment:
Do we need to consider adding `confirmation=True` for those remove commands and delete commands?

Reply:
We only added `confirmation=True` for deleting top level resources, thoughts on adding this for deleting non-top level resources? @anthonychu

Reply:
I'm okay without confirmation for these commands. But if CLI requires it that's fine